target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -182,7 +182,7 @@ ramips_setup_interfaces()
 	meig,slt866)
 		ucidef_set_interfaces_lan_wan "lan" "wan"
 		;;
-	*)
```

Comment:
this part looks wrong, please remove it, the patch should be good to merge once that change is made.

Reply:
oh, that does break things.